roles/docker-compose/tasks/container.yml

```diff
@@ -0,0 +1,256 @@
+    exposed_ports:
+      - "8008"
+    volumes:
+      - /root/synapse:/data
+      - /root/matrix-appservice-irc/appservice-registration-irc.yaml:/irc/appservice-registration-irc.yaml
+    labels:
+      traefik.enable: "true"
+      traefik.http.routers.synapse_http.rule: "Host(`matrix.{{ base_url }}`) && PathPrefix(`/_matrix`)"
+      traefik.http.routers.synapse_http.entryPoints: "http"
+      traefik.http.routers.synapse_http.middlewares: "https_redirect@file"
+      traefik.http.routers.synapse_https.rule: "Host(`matrix.{{ base_url }}`) && PathPrefix(`/_matrix`)"
+      traefik.http.routers.synapse_https.priority: "20"
+      traefik.http.routers.synapse_https.entryPoints: "https"
+      traefik.http.routers.synapse_https.tls.certresolver: "le_http"
+      traefik.http.services.synapse-app.loadbalancer.server.port: "8008"
+    state: stopped
+
+- name: Stop riot docker container
+  docker_container:
+    name: "riot"
+    image: "vectorim/riot-web"
+    restart_policy: unless-stopped
+    exposed_ports:
+      - "80"
+    volumes:
+      - /root/riot/config.json:/app/config.json
+    labels:
+      traefik.enable: "true"
+      traefik.http.routers.riot_http.rule: "Host(`chat.{{ base_url }}`)"
+      traefik.http.routers.riot_http.entryPoints: "http"
+      traefik.http.routers.riot_http.middlewares: "https_redirect@file"
+      traefik.http.routers.riot_https.rule: "Host(`chat.{{ base_url }}`)"
+      traefik.http.routers.riot_https.priority: "10"
+      traefik.http.routers.riot_https.entryPoints: "https"
+      traefik.http.routers.riot_https.tls.certresolver: "le_http"
+      traefik.http.services.riot-app.loadbalancer.server.port: "80"
+    state: stopped
+
+- name: Stop synapse-pgadmin docker container
+  docker_container:
+    name: "synapse-pgadmin"
+    image: "dpage/pgadmin4:latest"
+    restart_policy: unless-stopped
+    exposed_ports:
+      - "80"
+    env:
+      PGADMIN_DEFAULT_EMAIL: "ca-pgadmin@noemis.me"
+      PGADMIN_DEFAULT_PASSWORD: "{{ pgadmin_password }}"
+    labels:
+      traefik.enable: "true"
+      traefik.http.routers.pgadmin_http.rule: "Host(`pgadmin.{{ base_url }}`)"
+      traefik.http.routers.pgadmin_http.entryPoints: "http"
+      traefik.http.routers.pgadmin_http.middlewares: "https_redirect@file"
+      traefik.http.routers.pgadmin_https.rule: "Host(`pgadmin.{{ base_url }}`)"
+      traefik.http.routers.pgadmin_https.entryPoints: "https"
+      traefik.http.routers.pgadmin_https.tls.certresolver: "le_http"
+      traefik.http.services.pgadmin-app.loadbalancer.server.port: "80"
+    state: stopped
+
+- name: Create/Start matrix-appservice-irc docker container
+  docker_container:
+    name: "matrix-appservice-irc"
+    image: "tulir/matrix-appservice-irc:latest"
+    restart_policy: unless-stopped
+    command: "node app.js -c config.yaml -p 9995 -f appservice-registration-irc.yaml -u http://0.0.0.0:9995"
+    exposed_ports:
+      - "9995"
+    volumes:
+      - root_irc-appservice:/app/data
```

Comment:
Is this volume path correct? Does the data need to be moved?

Reply:
no because this it the only service currently remaining in docker. We didn't got it set up correctly.

Reply:
I saw that in the docker compose setup the volume was set as "irc-appservice:/app/data" and was wondering if the change to "root_" comes from not using compose anymore?

Reply:
the volume is from the the old setup, where the volumes sare named with the prefix root_